partner_create_by_vat/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================================
+Automatic partner creation based on VAT number
+==============================================
+
+This module allows you to create the partners (companies) based on their VAT number.
+Name and address of the partner will automatically be completed via VIES Webservice.
+
+VIES Service (based on stdnum python)
+http://ec.europa.eu/taxation_customs/vies/vieshome.do
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/partner-contact
```

Comment:
Usually better to use enumerated list for procedure (non blocking for me though)